src/core/sock.c

```diff
@@ -503,6 +503,9 @@ int nn_sock_add_ep (struct nn_sock *self, const struct nn_transport *transport,
 
     /*  Instantiate the endpoint. */
     ep = nn_alloc (sizeof (struct nn_ep), "endpoint");
+    if (!ep){
```

Comment:
nit: the style here does not match.  (whitespace between ) and {

Reply:
thanks @gdamore! I added whitespace!